test/suites/launchTemplates/suite_test.go

```diff
@@ -0,0 +1,93 @@
+package launchtemplates
+
+import (
+	"strings"
+	"testing"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/aws/karpenter/pkg/apis/provisioning/v1alpha5"
+	"github.com/aws/karpenter/pkg/cloudprovider/aws/apis/v1alpha1"
+	"github.com/aws/karpenter/pkg/test"
+
+	"github.com/aws/karpenter/test/pkg/environment"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var env *environment.Environment
+
+func TestLaunchTemplates(t *testing.T) {
+	RegisterFailHandler(Fail)
+	BeforeSuite(func() {
+		var err error
+		env, err = environment.NewEnvironment(t)
+		Expect(err).ToNot(HaveOccurred())
+	})
+	RunSpecs(t, "LaunchTemplates")
+}
+
+var _ = BeforeEach(func() {
```

Comment:
```suggestion
var _ = BeforeEach(env.BeforeEach)
```

Reply:
Didn't spend too long to debug, but this was making our tests fail so left it out.